server/views/pages/prisons/viewSessionTemplates.njk

```diff
@@ -126,7 +126,7 @@
 
               {%- set rows = (rows.push([
                 {
-                  html: '<a href="sessionTemplate/' + sessionTemplate.reference + '">' + sessionTemplate.name + "</a>",
+                  html: '<a href="session-templates/' + sessionTemplate.reference + '">' + sessionTemplate.name + "</a>",
```

Comment:
This works but would probably be better explicitly using the full path; e.g.:
`href="/prisons/' + prison.code + '/session-templates/....`

(I can see that this crept in from my previous PR 😳!)